utils/airtable_es2015.js

```diff
@@ -72,20 +95,7 @@ var hydrateFromAirtable = (exports.hydrateFromAirtable = async function hydrateF
   await Promise.all(promises);
   dataStore["errors"] = [];
   airtableConstants.tableNames.forEach(function(tableName) {
-    var array = dataStore[tableName].map(x => Object.keys(x).length);
```

Comment:
I removed this because it was kind of a dumb algorithm that sometimes resulted in unexpected consequences. What it did: if less than half of the columns in a row are filled, that row is rejected and an error is sent to airtable.